.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
```

Comment:
This is not the configuration that is requested. The content of this file is already provided in the ticket description